stdlib/email/feedparser.pyi

```diff
@@ -15,10 +15,10 @@ class FeedParser(Generic[_MessageT]):
     def feed(self, data: str) -> None: ...
     def close(self) -> _MessageT: ...
 
-class BytesFeedParser(Generic[_MessageT]):
+class BytesFeedParser(FeedParser[_MessageT]):
     @overload
     def __init__(self: BytesFeedParser[Message], _factory: None = None, *, policy: Policy = ...) -> None: ...
     @overload
     def __init__(self, _factory: Callable[[], _MessageT], *, policy: Policy = ...) -> None: ...
-    def feed(self, data: bytes | bytearray) -> None: ...
+    def feed(self, data: bytes | bytearray) -> None: ...  # type: ignore[override]
     def close(self) -> _MessageT: ...
```

Comment:
I think we need to keep the redefinition of __init__ even though it's not actually redefined in the subclass in cpython, due to the typing on the self argument.

That said, the redefinition of close() here could be removed. it's identical to that of the superclass. I wasn't sure whether that meant that it should be removed or not. In the end it kept the diff very simple to leave it alone, so that's what I did.

Reply:
We generally remove redundant definitions in sub-classes, especially if they aren't defined in the implementation.

Reply:
Thanks - I removed it.